### ISSUE TITLE
examples of streams

### ISSUE THREAD

**Issue by tttp** [nkirby/node-twitterbot#2]
Hi,

I'm trying to build a bot that receives command either as DM or "@nameofthebot nameofcommand"
is this the stream action I need to use? do you have an example of a bot receiving orders that way?

Specifically, wondering if I can build several bot.listen that have the same Streaming API requests (all the "@nameofthebot") parameters but a different listenerFunction (check if the nameof the command +extra params is there) or if I need one bot.listen and do all the logic withing the listenerFunction

X+

**Comment by techieshark** [comment]
Hey @tttp I'm looking into this too. Did you end up figuring it out?

**Comment by tttp** [comment]
nope, i needed something simple and just started from lower level lib and added the bits i needed
